modules/grpc/1.62.1.bcr.1/source.json

```diff
@@ -0,0 +1,11 @@
+{
+    "integrity": "sha256-yfmubk1vQEZO6ZWL5AaAh4ge1qo34w0OZNQO17453QE=",
+    "strip_prefix": "grpc-1.62.1",
+    "url": "https://github.com/grpc/grpc/archive/refs/tags/v1.62.1.tar.gz",
+    "patch_strip": 1,
+    "patches": {
+        "add_module_bazel.patch": "sha256-QA+8LXhdL6PhmaaQVLj9YN8+I02Y3gtgrUJghnhbEpc=",
```

Comment:
Could you replace the unmodified patches with relative paths a la `../../1.62.1/patches/...`? That will make the diff much smaller.

Reply:
How about the version change? Should I add an additional patch to change the version in `MODULE.bazel` then?

Reply:
Ah, forgot about that. Your call, but please use this trick for the large other patch.

Reply:
I added the extra patch and fixed the `update_integrity` tool along the way in https://github.com/bazelbuild/bazel-central-registry/pull/2163.